pkg/api/types.go

```diff
@@ -788,6 +788,14 @@ type NodeSpec struct {
 	ExternalID string `json:"externalID,omitempty"`
 }
 
+// NodeIdentity is a set of ids/uuids to uniquely identify the node.
+type NodeIdentity struct {
```

Comment:
We'll likely want more fields from cAdvisor's MachineInfo and VersionInfo structs eventually. How about we call this NodeSystemInfo, inline it into NodeInfo, and call the field NodeInfo in NodeStatus. When /nodeinfo is eliminated, we can rename NodeSystemInfo to NodeInfo.